src/sonic-config-engine/portconfig.py

```diff
@@ -72,12 +73,12 @@ def db_connect_configdb(namespace=None):
     """
     Connect to configdb
     """
-    try:
-        if namespace is not None:
+    if not swsscommon.SonicDBConfig.isInit():
+        if multi_asic.is_multi_asic():
             swsscommon.SonicDBConfig.load_sonic_global_db_config(namespace=namespace)
-        config_db = swsscommon.ConfigDBConnector(use_unix_socket_path=True, namespace=namespace)
-    except Exception as e:
```

Comment:
>Exception [](http://example.com/codeflow?start=11&length=9)

Could you use this PR for first issue in your description? And separate the second issue into another PR. The 2nd one is still under discussion.